SQL/New_patches/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,174 @@
+SET FOREIGN_KEY_CHECKS=0;
+-- Publication Status
+DROP TABLE IF EXISTS publication_status;
+CREATE TABLE `publication_status` (
+  `PublicationStatusID` int(2) unsigned NOT NULL AUTO_INCREMENT,
+  `Label` varchar(255) NOT NULL,
+  CONSTRAINT `PK_publication_status` PRIMARY KEY(`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+INSERT INTO publication_status (`Label`) VALUES ('Pending');
+INSERT INTO publication_status (`Label`) VALUES ('Approved');
```

Comment:
Publication status "Approved" strongly suggests that a manuscript was approved/accepted for publication by a publisher. 

If this means just "Project/Access approved" - then consider renaming